CONTRIBUTING.md

```diff
@@ -218,7 +218,7 @@ For a pull request to be considered at all it has to meet these requirements:
 
 Some additional guidelines regarding source code are:
 
-- Files should start with a ``Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>`` copyright header.
+- Files should start with a ``Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>`` copyright header.
```

Comment:
I think we should switch to https for the URL

Reply:
i'll update it